.prettierrc

```
@@ -0,0 +1,9 @@
+{
+  "endOfLine": "lf",
+  "semi": true,
+  "singleQuote": true,
```

Let's change this to false

```suggestion
  "singleQuote": false,
```